include/dlaf/eigensolver/tridiag_solver/merge.h

```diff
@@ -291,24 +303,15 @@ struct TileCollector {
 };
 
 // Note: not using matrix::copy for Tile<> here because this has to work for U = SizeType too.
-template <class U>
-void copyVector(SizeType i_begin, SizeType i_end, Matrix<const U, Device::CPU>& in,
-                Matrix<U, Device::CPU>& out) {
+template <Backend backend, Device device, class U>
+void copyVector(SizeType i_begin, SizeType i_end, Matrix<const U, device>& in, Matrix<U, device>& out) {
   namespace ex = pika::execution::experimental;
   namespace di = dlaf::internal;
 
-  auto copy_fn = [](const matrix::Tile<const U, Device::CPU>& in,
-                    const matrix::Tile<U, Device::CPU>& out) {
-    SizeType rows = in.size().rows();
-    for (SizeType i = 0; i < rows; ++i) {
-      TileElementIndex idx(i, 0);
-      out(idx) = in(idx);
-    }
-  };
   for (SizeType i = i_begin; i <= i_end; ++i) {
     GlobalTileIndex idx(i, 0);
     ex::when_all(in.read_sender(idx), out.readwrite_sender(idx)) |
-        di::transform(di::Policy<Backend::MC>(), copy_fn) | ex::start_detached();
+        di::transform(di::Policy<backend>(), matrix::internal::copy_o) | ex::start_detached();
```

Comment:
```suggestion
        dlaf::tile::copy(di::Policy<CopyBackend_v<...>>()) | ex::start_detached();
```
? Or does that not work?